docs/learn/smart-contract-developers/create-lsp7-token.md

```diff
@@ -226,234 +211,12 @@ main()
-
-</Tabs>
-
-### Send transaction
-
-Finally, deploy the contract.
-
-<Tabs>
-  
-  <TabItem value="web3js" label="web3.js">
-
-<!-- prettier-ignore-start -->
-
-```javascript title="Deploy the LSP7 Digital Asset contract"
-await myToken.deploy({
-    data: LSP7Mintable.bytecode,
-    arguments: [
-      'My LSP7 Token', // token name
-      'LSP7',          // token symbol
-      account.address, // new owner, who will mint later
-      false,           // isNonDivisible = TRUE, means NOT divisible, decimals = 0)
-    ],
-  })
-  .send({ from: account.address });
-```
-<!-- prettier-ignore-end -->
-
-  </TabItem>
-
-<!-- prettier-ignore-start -->
-
-  <TabItem value="ethersjs" label="ethers.js">
-
-```javascript title="Deploy the LSP7 Digital Asset contract"
-const myToken = await lsp7Factory.connect(myEOA).deploy(
-  'My LSP7 Token', // token name
-  'LSP7',          // token symbol
-  myEOA.address,   // new owner, who will mint later
-  false,           // isNonDivisible = TRUE, means NOT divisible, decimals = 0)
-);
-```
-
-
-<!-- prettier-ignore-end -->
-
-  </TabItem>
-
-</Tabs>
-
-### Final code
-
-<!-- prettier-ignore-start -->
-
-<Tabs>
-  
-  <TabItem value="web3js" label="web3.js">
-
-```javascript
-import LSP7Mintable from '@lukso/lsp-smart-contracts/artifacts/LSP7Mintable.json';
-import Web3 from 'web3';
-
-const web3 = new Web3('https://rpc.testnet.lukso.network');
-
-// initialize your EOA
-const privateKey = '0x...';
-const account = web3.eth.accounts.wallet.add(privateKey);
-
-// create a contract instance
-const myToken = new web3.eth.Contract(LSP7Mintable.abi, {
-  gas: 5_000_000,
-  gasPrice: '1000000000',
-});
-
-// deploy the token contract
-await myToken.deploy({
-    data: LSP7Mintable.bytecode,
-    arguments: [
-      'My LSP7 Token', // token name
-      'LSP7',          // token symbol
-      account.address, // new owner, who will mint later
-      false,           // isNonDivisible = TRUE, means NOT divisible, decimals = 0)
-    ],
-  })
-  .send({ from: account.address });
-```
-
-  </TabItem>
-
-  <TabItem value="ethersjs" label="ethers.js">
-
-```javascript
-import LSP7Mintable from '@lukso/lsp-smart-contracts/artifacts/LSP7Mintable.json';
-import { ethers } from 'ethers';
-
-const provider = new ethers.providers.JsonRpcProvider('https://rpc.testnet.lukso.network');
-
-// set up your EOA
-const privateKey = '0x...';
-const myEOA = new ethers.Wallet(privateKey).connect(provider);
-
-// create an instance of the token contract
-const lsp7Factory = new ethers.ContractFactory(
-  LSP7Mintable.abi,
-  LSP7Mintable.bytecode,
-);
-
-// deploy the token contract
-const myToken = await lsp7Factory.connect(myEOA).deploy(
-  'My LSP7 Token', // token name
-  'LSP7',          // token symbol
-  myEOA.address,   // new owner, who will mint later
-  false,           // isNonDivisible = TRUE, means NOT divisible, decimals = 0)
-);
-```
-
-  </TabItem>
-
-</Tabs>
+## Check your token address on the Execution Block Explorer
```

Comment:
Putting a new `h2` heading just for one sentence is unnecessary. I would suggest removing it and putting the sentence within `deployment`